### PR TITLE
Fix line breaks in RSS feed

### PR DESCRIPTION
**Pull request by nickvergessen**
Required for multi line message bodies

### CODE REVIEW

File: templates/rss.php [hunk 1]
```diff
@@ -30,7 +30,7 @@
 			<pubDate><?php p(date('r', $activity['timestamp'])); ?></pubDate>
 <?php endif; ?>
 <?php if (!empty($activity['message'])): ?>
-			<description><![CDATA[<?php p($activity['messageformatted']['full']); ?>]]></description>
+			<description><![CDATA[<?php print_unescaped(str_replace("\n", '<br />', OC_Util::sanitizeHTML($activity['messageformatted']['full']))); ?>]]></description>
```
Comment (nickvergessen) [inline templates/rss.php]: @LukasReschke please approve

Comment (LukasReschke) [inline templates/rss.php]: FIX IT

Comment (nickvergessen) [inline templates/rss.php]: FIXED IT!

**Comment by LukasReschke** [comment]
:+1:

**Pull request merged**